python/ray/tests/test_kill_subprocesses.py

```diff
@@ -139,7 +139,6 @@ def auto_reap(self):
             pid = process.pid
             time.sleep(1)  # wait for the process to exit.
```

Comment:
can you remove this and convert it to use `wait_for_condition`?

Reply:
good idea, updated